### PR TITLE
Migration for services, guides, step by step, topics and users

### PR DESCRIPTION
**Pull request by stephen-cox**
Checks migrations Ekes wrote still work and adds user migrations.

### CODE REVIEW

**Comment by stephen-cox** [comment]
This is an initial migration. It should look pretty similar to @andybroomfield as there's only been a few tweaks and the addition of users to what he'd already done.

I have deployed and run migrations on the BHCC and Croydon test sites:
- https://bhcc.localgov.agile.coop/
- https://croydon.localgov.agile.coop/

@Adnan-cds @andybroomfield Can you have a look at the content migration on these sites and let me know if there are any issues that haven't already been recorded? You should be able to login with Drupal user 1 for your site, but let me know in Slack if you'd like me to reset the password for this account.

If you want to run the migrations locally, create a new database and add Drupal settings for this using the `migration` array key. Then install the module with `composer require localgovdrupal/localgov_legacy_migration`, check out the right branch and enable the module. The migrations are all in the `localgov_migration` group so they can all be run with `drush mim --group=localgov_migration`

**Review by andybroomfield — APPROVED** [review]
Migrate worked and users where correctly attributed to their content.

File: localgov_legacy_migration.module [hunk 1]
```diff
@@ -5,7 +5,7 @@ use Drupal\migrate\Plugin\MigrationInterface;
 use Drupal\migrate\Row;
 
 function localgov_legacy_migration_migrate_prepare_row(Row $row, MigrateSourceInterface $source, MigrationInterface $migration) {
-  print $migration->id();
-
-  print_r($row->getSource());
+//  print $migration->id();
+//
+//  print_r($row->getSource());
 }
```
Comment (andybroomfield) [inline localgov_legacy_migration.module:11]: Better to simply remove this function?

**Comment by willguv** [comment]
Nice work everyone!!

Is this something we can ask content designers to check? I’m back working on Monday so we can discuss at standup

**Comment by Adnan-cds** [comment]
Hi Will,
Yes, please ask content designers to take a look.  I was going to ask you anyway :)

FAO @willguv

**Comment by stephen-cox** [comment]
Thanks for taking a look. The migrations will continue to evolve in a number of pull requests, so I'm merging this one. 

Next up files and media.

**Pull request merged**